### PR TITLE
Keep windows from accumulating when resetting the plugin

### PR DESCRIPTION
**Pull request by eggrobin**
I hate garbage-collected languages.

### CODE REVIEW

File: ksp_plugin_adapter/window_renderer.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      }
+      all_ = value;
+    }
+  }
+
+  private T all_;
+}
+
+}  // namespace principia
+}  // namespace ksp_plugin_adapter
```
Comment (pleroy) [inline ksp_plugin_adapter/window_renderer.cs]: Bad order.

File: ksp_plugin_adapter/window_renderer.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+  abstract protected void RenderWindow();
+
+  private ManagerInterface manager_;
+}
+
+internal struct Controlled<T> where T : IDisposable {
+  public T all {
+    get { return all_; }
```
Comment (pleroy) [inline ksp_plugin_adapter/window_renderer.cs]: Return statement on its own line.

File: ksp_plugin_adapter/window_renderer.cs (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin_adapter/window_renderer.cs]: Done.

File: ksp_plugin_adapter/window_renderer.cs (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_adapter/window_renderer.cs]: Done.

**Pull request merged**